packages/server-wallet/CHANGELOG.md

```diff
@@ -3,6 +3,105 @@
+* remove required chainServiceRequests property from Channel model and fix tests ([d20eae8](https://github.com/statechannels/statechannels/commit/d20eae80ce48dc53d013cecba95036ce6a28b846))
+* remove stale comment ([139f4be](https://github.com/statechannels/statechannels/commit/139f4be77e73c8e7bda91dfc428dca1ef021ccc5))
+* remove unneeded await ([c2db964](https://github.com/statechannels/statechannels/commit/c2db964993a5e49c22cda2693da99e23d2559325))
+* remove unneeded type conversion ([d753b8d](https://github.com/statechannels/statechannels/commit/d753b8d3027e56a3378f7bff252b87a18c1e508d))
+* return finalizesAt on challenge active ([2a878cc](https://github.com/statechannels/statechannels/commit/2a878ccdf92f84b68ec3825fcc38ec9c570a6ff6))
+* set chain id ([d1877c8](https://github.com/statechannels/statechannels/commit/d1877c868d9c091b66dca8ccc35373062b720a04))
+* simply logic by removing status ([3a9fdf5](https://github.com/statechannels/statechannels/commit/3a9fdf5ba2b80bc3f1772448650413a3b546fb86))
+* standardize chain service public method logging ([10c4d75](https://github.com/statechannels/statechannels/commit/10c4d751fb4211d267d3f33cb445d137434f90c7))
+* support fake funding in directFundingStatus ([6b1c2d7](https://github.com/statechannels/statechannels/commit/6b1c2d76960823e835971dc4f2b12c4803c47293))
+* update wallet response to use queuedMessages over outbox property for queued ledger proposal ([2c6fd25](https://github.com/statechannels/statechannels/commit/2c6fd25d567938f61ad0b93461339ddfc0797350))
+* use ?? instead of || for defaulting blockConfirmation ([94aaf0f](https://github.com/statechannels/statechannels/commit/94aaf0fa2b7f5bd7af4166351c614f125eff7953))
+* use distinct database names ([57894fb](https://github.com/statechannels/statechannels/commit/57894fbe8c82f2d845af929a73b6a53afc3fc5b7))
+* use pqueue to avoid onBlockMined race condition ([cd52339](https://github.com/statechannels/statechannels/commit/cd523397821c5e1852e02cbe946a7c8e9e259a6a))
+* use raw NULL for commit column ([1bc1a77](https://github.com/statechannels/statechannels/commit/1bc1a774a55891affcccaaf3d5252a49903f324e))
+* user separate address for test adjudicator ([e837eaa](https://github.com/statechannels/statechannels/commit/e837eaa661bb1c3814bd3188f3fa6753cc03d4e5))
+* validate metrics ([2e6fc84](https://github.com/statechannels/statechannels/commit/2e6fc841e4a13cd4c8705e094d49bb1ad20f8f03))
+* wait for all updateTransferredOut before taking actions ([e3eb6ca](https://github.com/statechannels/statechannels/commit/e3eb6caf4a39f5b0a00157537cd6d8a30be6ca21))
+
+
+### Features
+
+* add a mocked out chain service channelFinalized API ([2e31d92](https://github.com/statechannels/statechannels/commit/2e31d92f8e0997cf9ca9924ff6f38d2ba9530dd4))
+* add chain transactions table ([4819b87](https://github.com/statechannels/statechannels/commit/4819b87667b4c05b843c0033a3b988f9cc697493))
```

Comment:
Is this a feature? It seems like adding a table isn't enough to implement a feature.